presto-main/src/main/resources/webapp/src/components/QueryList.jsx

```diff
@@ -63,13 +63,23 @@ export class QueryListItem extends React.Component {
         }
 
         let formattedQueryText = "";
+        let currentLine = 0;
 
         for (let i = 0; i < lines.length; i++) {
             const trimmedLine = lines[i].substring(minLeadingWhitespace).replace(/\s+$/g, '');
+            currentLine += Math.ceil(trimmedLine.length / 75);
```

Comment:
Is 75 the max width per line? If so, let's create a constant `const maxWidthPerLine = 75` and replace the hard-coded `75` with it.

Reply:
done